config/ant-phase-verify.xml

```diff
@@ -12,7 +12,6 @@
              classpath="${mvn.runtime_classpath}"
              />
 
-    <property name="checkstyle.pattern.todo" value="NOTHingWillMatCH_-"/>
```

Comment:
Been around for 12 years at 86cbe5bd160285d46535c03e07a197fc542727ce but never been used.